src/routes/sessions/[id]/Prompt.svelte

```diff
@@ -45,14 +63,42 @@
 		if (event.shiftKey) return;
 		if (event.key !== 'Enter') return;
 		event.preventDefault();
+		submit();
+	}
+
+	function handleSelectKnowledge(fieldId: string, knowledgeId: string) {
+		$attachments = $attachments.map((a) =>
+			a.fieldId === fieldId ? { ...a, knowledge: loadKnowledge(knowledgeId) } : a
+		);
+	}
+
+	function handleDeleteAttachment(fieldId: string) {
+		$attachments = $attachments.filter((a) => a.fieldId !== fieldId);
+	}
+
+	function submit() {
+		if ($attachments.length) {
+			const attachmentMessages: Message[] = [];
+			$attachments.forEach((a) => {
+				if (a.knowledge)
+					attachmentMessages.push({
+						role: 'user',
+						knowledge: a.knowledge,
+						content: a.knowledge.content
```

Comment:
Let's add some formatting to the `content` value so the LLM understands what this text means

```jsx
`
CONTEXT
---
${a.knowledge.name}
---
${a.knowledge.content}
`
```

Reply:
I'm sorry, I'm afraid I don't understand this syntax. I understand the context problem though, so where should we make the change?

Reply:
Sorry, I meant that `content: a.knowledge.content` should be an interpolated string that includes the word `CONTEXT`, then the knowledge `name` and finally the `content` separated by `---`:

```typescript
attachmentMessages.push({
  role: 'user',
  knowledge: a.knowledge,
  content: `
CONTEXT
---
${a.knowledge.name}
---
${a.knowledge.content}
`})
```

Reply:
Thanks! All clear now.